lib/action_cable/connection/base.rb

```diff
@@ -165,6 +165,28 @@ def on_close
         end
 
 
+        def allow_request_origin?
+          return true if server.config.disable_request_forgery_protection
```

Comment:
Arguably we should never disable this. Instead we can get a future feature that lets someone do a wildcard Origin whitelist match.